index.d.ts

```diff
@@ -359,6 +359,34 @@ export declare interface TracerOptions {
      * @default false
      */
     enableGetRumData?: boolean
+
+    /**
+     * Configuration of the IAST. Can be a boolean as an alias to `iast.enabled`.
+     */
+    iast?: boolean  | {
+      /**
+       * Whether to enable IAST.
+       * @default false
+       */
+      enabled?: boolean,
+      oce?: {
```

Comment:
Is this acronym common enough so that users will know what it means? A quick google search returned nothing.

Reply:
I have talk about it with java team also, it looks like we are going to remove `oce` concept from code, and use directly: `iast.requestSampling` and `process.env.DD_IAST_REQUEST_SAMPLING` without the `oce`.

Reply:
Just out of curiosity, what does OCE mean in the first place?

Reply:
Overhead Controller Engine 😄